plugins/dashboards/frontend/public/javascripts/countly.views.js

```diff
@@ -1340,91 +1118,6 @@
                     this.updateWidgetGeography(wId, {size: size, position: position});
                 }
             },
-            autoPosition: function(allWidgets) {
```

Comment:
@kanwarujjaval do you think we'll need something to handle customers who are already using the new UI?

Reply:
I don't think there is a direct way of doing that since,
In the 4 column grid system, total sum of x coordinates of all widgets can be maximum 6, but now we are reverting to 12 columns, so there is not definite way of telling if it is old ui, or the new UI.
Will test some cases and get back to you